src/shared/components/query/CancerStudyTreeData.ts

```diff
@@ -63,14 +64,20 @@ export default class CancerStudyTreeData
 		const virtualCohortStudies = [];
 		for (let virtualCohort of virtualCohorts) {
 			let study = {
-				allSampleCount: virtualCohort.samples.length,
+				//allSampleCount: [].concat.apply([], virtualCohort.data.studies.map(study => study.samples)).length,
+				allSampleCount:virtualCohort.data.studies
+													.map(study => study.samples)
+													.reduce(function(prev, curr) {
+														return prev.concat(curr);
+													})
+													.length,
 				studyId: virtualCohort.id,
-				name: virtualCohort.name,
-				description: virtualCohort.description,
+				name: virtualCohort.data.name,
+				description: virtualCohort.data.description,
 				cancerTypeId: virtualCohortsName
 			} as CancerStudy;

```

Comment:
if at all possible, avoid using "as" because actually then lose type checking on this object.  try to actually construct a study that satisfies typing.   then you can do const study: CancerStudy = {completes study}